src/daemon/rpmostreed-daemon.c

```diff
@@ -313,6 +312,23 @@ maybe_load_config_keyfile (GKeyFile **out_keyfile,
   return TRUE;
 }
 
+static guint64
+get_config_uint64 (GKeyFile   *keyfile,
+                   const char *key,
+                   guint64     default_val)
+{
+  if (keyfile && g_key_file_has_key (keyfile, DAEMON_CONFIG_GROUP, key, NULL))
+    {
+      g_autoptr(GError) local_error = NULL;
+      guint64 r = g_key_file_get_uint64 (keyfile, DAEMON_CONFIG_GROUP, key, &local_error);
+      if (!local_error)
+        return r;
+      sd_journal_print (LOG_WARNING, "Bad value for key '%s': %s; using compiled defaults",
```

Comment:
Random side question; does using `sd_journal_print()` in this situation give us anything over just `g_printerr()`?

Reply:
Not as far as I can tell. Though the rest of the file also uses `sd_journal_print`, so might as well be consistent I guess?